prompting/validators/event.py

```diff
@@ -137,6 +137,35 @@ def from_dict(event_dict: dict, disable_log_rewards: bool) -> "EventSchema":
                 RewardModelType.prompt.value + "_normalized"
             ),
         }
+        penalties = {
```

Comment:
this just for experiments or you want to go into prod with all these?

Reply:
I am still on doubt, I was thinking to delete one between the `adjusted` and `applied`, but not sure yet. What's your thoughts on this?

Reply:
fine while we collect data. we can take it out next week maybe